src/main/java/rpgcharacters/userflow/CreateCharacterMenu.java

```diff
@@ -68,40 +91,60 @@ private int getStat(String statName, int curStatVal) {
             statMod = sc.nextInt();
         }
         curMaxAdd -= statMod;
-        return curStatVal + statMod;
+        return statMod;
     }
 
-    private boolean createCharacter () {
+    private ResultSet getStatRow(String table, String name) throws SQLException {
+        String query = "SELECT * FROM " + table
+                     + " WHERE name='" + name.replaceAll("'", "''") + "';";
+        Statement stmt = conn.createStatement();
+        ResultSet results = stmt.executeQuery(query);
+        if (!results.last()) {
+            System.out.println("\n" + name + " does not exist.");
+            return null;
+        }
+        return results;
+    }
 
-        ArrayList<String> charAttributes = new ArrayList<String>();
+    private boolean createCharacter() {
 
         System.out.print("Character Name: ");
         String charName = sc.nextLine();
 
         System.out.print("Character Backstory: ");
         String charStory = sc.nextLine();
 
-        System.out.print("Character Race: ");
-        String charRace = sc.nextLine();
-
-        System.out.print("Character Archetype: ");
-        String charArch = sc.nextLine();
-
-        int[] curStats = getCurStats(charRace,charArch);
-
-        curMaxAdd = 20;
-        curStats[0] = getStat("power",curStats[0]);
-        curStats[1] = getStat("proficiency",curStats[1]);
-        curStats[2] = getStat("personality",curStats[2]);
-        curStats[3] = getStat("perception",curStats[3]);
-
-        return saveToDB(charName,charStory,charRace,charArch,curStats);
+        try {
+            System.out.print("Character Race: ");
```

Comment:
How does the user know what races are available? Replace this with a similar options menu to every other user selection (i.e. "1. Human")